app/src/main/java/com/example/roome/MyPreferences.java

```diff
@@ -16,8 +16,22 @@ static boolean isFirstTime(Context context) {
         return reader.getBoolean(IS_FIRST_TIME, true);
     }
 
+    static void setIsFirstTimeToFalse(Context context) {
+        final SharedPreferences reader = context.getSharedPreferences(MY_PREFERENCES, Context.MODE_PRIVATE);
+        final SharedPreferences.Editor preferencesEditor = reader.edit();
+        preferencesEditor.putBoolean(MyPreferences.IS_FIRST_TIME, false);
+        preferencesEditor.apply();
+    }
+
+    static void setIsRoommateSearcherToFalse(Context context) {
+        final SharedPreferences reader = context.getSharedPreferences(MY_PREFERENCES, Context.MODE_PRIVATE);
+        final SharedPreferences.Editor preferencesEditor = reader.edit();
+        preferencesEditor.putBoolean(MyPreferences.IS_ROMMATE_SEARCHER, false);
+        preferencesEditor.apply();
+    }
+
```

Comment:
מעולה!

Reply:
ישמצב שכדאי להוציא למחלקה את ה
reader and the editor
?

Reply:
מממ לא חושב